transport/util/errormapper.go

```diff
@@ -17,6 +17,8 @@ func ToResponseError(err error) *responseerrors.ResponseError {
 		return responseerrors.NewUnprocessableEntityError().Wrap(err)
 	case apierrors.IsConflictError(err):
 		return responseerrors.NewConflictError().Wrap(err)
+	case apierrors.IsPreconditionFailedError(err):
```

Comment:
> High-level comment - as indicated earlier, consider gathering transport layer error mapper and transport layer errors into one file / one package.
Ideally under transport layer.

Good point, will be addressed in https://github.com/jan-zabloudil/release-manager/issues/25

> Second comment - is it really a Precondition Failed error? From a quick read, this is a completely different kind of error... 🤔

Yes, you are right. If GitHub is not enabled or the GitHub repository is not configured, a 404 error is returned now (the resource is not found because it is not configured). Feel free to suggest a different response code if you think there is a better choice.